data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1951 @@
+				goto end
+			label accident
+			`	"A terrorist attack, it was. The valuable resources and equipment stockpiled there, they sought. Thanks to the agents' efforts though, robbed, the warehouses were not," she says, pointing to the injured Heliarchs being moved from your ship and into the land vehicles.`
+			label end
+			`	She bids you farewell and goes back to making sure the injured agents find their way to hospitals.`
+				accept
+
+mission "Heliarch Containment 3"
+	name "Informational Medicine"
+	description "While picking up <cargo> from <planet stopovers>, let loose the spy bots the Heliarchs gave you to act as surveillance for the planet."
+	cargo "medication" 15
+	to offer
+		has "Heliarch Containment 2: done"
+		not "Lunarium: Questions: done"
+	source "Ring of Power"
+	stopover "Secret Sky"
+	on offer
+		conversation
+			branch license
+				has "Heliarch License: done"
+			`Three Heliarch officers approach you as you're walking around the spaceport. "Greetings, Captain <last>. Interested, would you be, in accompanying us? A proposition, we have," the Saryd says.`
+			choice
+				`	"What's it about?"`
+				`	"Sure, lead the way."`
+					goto way
+				`	"Sorry, not up for taking a stroll right now."`
+					decline
+			`	"Inform you of that, we cannot," the Kimek says. "Not until accepted our offer, you have."`
+			choice
+				`	"Fine. Bring me to somewhere you can tell me."`
+				`	"That seems a bit sketchy, I'll pass."`
+					decline
+			label license
+			`Three Heliarch officers approach you as you're walking around the spaceport. They salute you, and the Saryd speaks. "Greetings, Captain <last>. Sent by Consul Plortub, we were."`
+			`	"A mission for you, he has, but to discuss it in private, he wishes to." The Kimek explains. "Follow us, would you?"`
+			label way
+			`	After a few dozen twists and turns through the restricted sections of the ring, they bring you to a room where an Arach is waiting, alongside a few other Heliarchs who are discussing something while pointing at a map of the Coalition projected over the table.`
+			`	"Ready to work with us, are you, Captain?" the Arach asks. "The one who asked for you, I am. Call me Consul Plortup, you may."`
+			choice
+				`	"<first> <last>. Pleasure to meet you."`
+				`	"What job did you have for me?"`
+			`	"In need of one such as you, we are. Anonymous reports, we have received, from loyal patriots and colleagues alike. On <stopovers>, a Saryd world, terrible activity may be brewing. Rising in activity there, outlaw groups are, we fear," he says. "Your destination, that is."`
+			`	A couple small boxes are brought inside and set on the table. Upon opening them, Plortup shows you they contain several small drones - drones very similar to those employed by the repair modules used here.`
```

Comment:
```suggestion
			`	A couple small boxes are brought inside and set on the table. Upon opening them, Plortup shows you they contain several small drones - drones very similar to those employed by the repair modules used here.`
```
Where is here? What for? Although I know what it's referencing, this seems a little out of left field.

Reply:
Changed to `...very similar to those employed by the hull repair modules used in Coalition ships.`